src/pages/library/Showcase.js

```diff
@@ -84,7 +85,7 @@ const Showcase = () => {
   const [fullList, setFullList] = useState(stateHistory.featuredWorkspaces)
 
   useOnMount(() => {
-    const loadData = async () => {
+    const loadData = withErrorReporting('Error loading showcase', async () => {
```

Comment:
UX says these error messages should end with a period.

Reply:
I posted on [Slack](https://broadinstitute.slack.com/archives/C34GAKQ5R/p1649855343820179?thread_ts=1648570756.312039&cid=C34GAKQ5R) to double-check with UX. We seem to have a ton of these messages with no full stop.

Reply:
I thought the outcome of the last [Slack discussion](https://broadinstitute.slack.com/archives/C34GAKQ5R/p1648570756312039) on punctuation was that these messages should end with a period.

Reply:
I've had a difficult time being sure myself.